src/chttpd/src/chttpd_auth_cache.erl

```diff
@@ -155,15 +155,17 @@ spawn_changes(Since) ->
     Pid.
 
 listen_for_changes(Since) ->
-    ensure_auth_ddoc_exists(dbname(), <<"_design/_auth">>),
+    DbName = dbname(),
+    erlang:put(io_priority, {system, DbName}),
```

Comment:
Would `ioq:set_io_priority({system, DbName})` work here (for consistency mainly)